deploy/helm/grafana-operator/templates/rbac.yaml

```
@@ -11,10 +11,11 @@ kind: {{ if not $namespaceScoped }}Cluster{{ end }}Role
 metadata:
   namespace: {{ . }}
   name: grafana-operator-permissions
-  {{- with (merge $.Values.additionalLabels (include "grafana-operator.labels" $ | fromYaml)) }}
   labels:
+    {{- include "grafana-operator.labels" . | nindent 4 }}
+    {{- with .Values.additionalLabels }}
```

I think you won't be able to render a chart with this (try with `helm template .`). When accessing .Values from a range, you need to use global context (`$`). Something like the changes below should help:

```suggestion
    {{- include "grafana-operator.labels" $ | nindent 4 }}
    {{- with $.Values.additionalLabels }}
```